tools/topology/topology1/development/sof-apl-nocodec-demux-eq-2ch4ch.m4

```diff
@@ -31,9 +31,9 @@ dnl I.e. row index is the input channel, 1 means it is copied to
 dnl corresponding output channel (column index), 0 means it is discarded.
 dnl There's a separate matrix for all outputs.
 define(matrix1, `ROUTE_MATRIX(1,
-			     `BITS_TO_BYTE(1, 0, 0 ,0 ,0 ,0 ,0 ,0)',
 			     `BITS_TO_BYTE(1, 0, 0 ,0 ,0 ,0 ,0 ,0)',
 			     `BITS_TO_BYTE(0, 1, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(1, 0, 0 ,0 ,0 ,0 ,0 ,0)',
```

Comment:
something's weird here. Why would you change this nocodec file when the routing changes you added are in the next patch?

Reply:
here we change the demux routing from L-L-R-R to L-R-L-R since we are going to change the default eq from lo-hi-lo-hi to lo-lo-hi-hi

Reply:
why? this is a nocodec example that shouldn't be touched by ODM-specific changes.

Reply:
Currently we ask odm to follow our channel order defined in the topology/machine driver so the only odm-specific thing should be their EQ coefficient. They need to generate it outside the sof repo.

Reply:
Not following @brentlu 

Someone else will have to review this PR.

@singalsu @juimonen can you please help?

Reply:
This was just an example, no issue in changing it.  I used it to test that the LLRR lhlh band-split version. Also this way we have vehicle (UP2) to test the topology.